### PR TITLE
Update sensitive-information-type-entity-definitions.md

### PR DESCRIPTION
**Pull request by pratjain523**
@chrfox Updates to the definition of 2 SITs. Also removing the message of partial availability of some SITs since now all solutions including endpoint have those SITs

### CODE REVIEW

**Comment by pratjain523** [comment]
@chrfox Added the definition of 2 new SITs as well

File: microsoft-365/compliance/sensitive-information-type-entity-definitions.md [hunk 1]
```diff
@@ -13882,6 +13648,69 @@ This unbundled named entity detects patterns related to physical address from No
 
 Medium
 
+## Philippines passport number 
+
+### Format
+
+Seven or eight or nine alphanumeric characters
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitive-information-type-entity-definitions.md]:
```suggestion
seven or eight or nine alphanumeric characters
```

File: microsoft-365/compliance/sensitive-information-type-entity-definitions.md [hunk 2]
```diff
@@ -13882,6 +13648,69 @@ This unbundled named entity detects patterns related to physical address from No
...
+
+- one letter followed by six digits.
+or
+- two letters followed by six digits. 
+or
+- two letters followed by seven digits. 
+or
+- one letter followed by seven digits followed by one letter
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitive-information-type-entity-definitions.md]:
```suggestion
- one letter followed by six digits
or
- two letters followed by six digits
or
- two letters followed by seven digits
or
- one letter followed by seven digits followed by one letter.
```

**Comment by yogkumgit** [comment]
@pratjain523 Please implement the suggested changes. Thanks.

File: microsoft-365/compliance/sensitive-information-type-entity-definitions.md [hunk 3]
```diff
@@ -13882,6 +13648,69 @@ This unbundled named entity detects patterns related to physical address from No
...
 
+## Philippines passport number 
+
+### Format
+
+Seven or eight or nine alphanumeric characters
+
+### Pattern
+
+Seven or eight or nine alphanumeric characters
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitive-information-type-entity-definitions.md]:
```suggestion
Seven or eight or nine alphanumeric characters:
```

File: microsoft-365/compliance/sensitive-information-type-entity-definitions.md [hunk 4]
```diff
@@ -14839,6 +14652,62 @@ A DLP policy has low confidence that it's detected this type of sensitive inform
...
 
+## Qatari ID card number
+
+### Format
+
+eleven digits
+
+### Pattern
+
+eleven digits:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitive-information-type-entity-definitions.md]:
```suggestion
Eleven digits:
```

File: microsoft-365/compliance/sensitive-information-type-entity-definitions.md [hunk 5]
```diff
@@ -14839,6 +14652,62 @@ A DLP policy has low confidence that it's detected this type of sensitive inform
...
+
+- a digit 2 or 3 
+- two digits representing the last two numbers of year of birth 
+- three digits representing ISO country code 
+- five digits
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitive-information-type-entity-definitions.md]:
```suggestion
- five digits.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @pratjain523, kindly commit the minor suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@pratjain523 Please implement the suggested changes. Thanks

**Comment by yogkumgit** [comment]
@pratjain523 Gentle reminder

**Comment by pratjain523** [comment]
Committed all suggestions.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Comment by chrfox** [comment]
holding this until after the publication of split SIT entity defn

**Comment by pratjain523** [comment]
@chrfox I see the doc has been split. Can you help push these changes through?

**Comment by chrfox** [comment]
@pratjain523  - You can to make these changes to the individual articles now, they are all public.

**Comment by chrfox** [comment]
all accomplished in other PR's

**Pull request closed without merging**